doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -172,14 +172,21 @@ include::../api/protos/vkCreateInstance.txt[]
   * pname:pInstance points a sname:VkInstance handle in which the resulting
     instance is returned.
 
-fname:vkCreateInstance creates the instance, then enables and initializes
-global layers and extensions requested by the application.
-If an extension is provided by a layer, both the layer and extension must:
-be specified at fname:vkCreateInstance time.
-If a specified layer cannot be found, no sname:VkInstance will be created
-and the function will return ename:VK_ERROR_LAYER_NOT_PRESENT.
-Likewise, if a specified extension cannot be found the call will return
+fname:vkCreateInstance verifies that the requested layers exist. If not,
+fname:vkCreateInstance will return ename:VK_ERROR_LAYER_NOT_PRESENT.
+Next fname:vkCreateInstance verifies that the requested extensions are
+supported (e.g. in implementation or enabled instance layers) and if
+any requested extension is not supported by the implementation or an
+enabled layer, fname:vkCreateInstance must: return
 ename:VK_ERROR_EXTENSION_NOT_PRESENT.
+After verifying and enabling the instance layers and extensions the
+sname:VkInstance object is created and returned to the application.
+If a requested extension is only supported by a layer, both the layer and
+extension need to be specified at fname:vkCreateInstance time.
+For example, if an application wants to enable extensionB that is only
+supported in layerB, the application needs to include "layerB" in
+sname:ppEnabledLayerNames and "extensionB" in the
+sname:ppEnabledExtensionNames list.
```

Comment:
- This whole thing above renders as a single large monolithic paragraph -> unesthetical and hard to read.
 - If I understand things, the layer supported extension case description is now redundant to the descriptions above. It could be a Note block instead (would also help with the big paragraph problem)